lib/assets/notification.css

```diff
@@ -151,3 +151,10 @@ html, body {
   background: #147EFB;
   color: #fff;
 }
+.flat, .flat:before,
+#notification.flat, #notification.flat:before,
+#notification #buttons.flat, #notification #buttons.flat:before,
+#notification .icon.flat, #notification .icon.flat:before,
+#notification .message.flat, #notification .message.flat:before  {
+  border-radius: 0px;
```

Comment:
This feels redundant. Wouldn't line 154 be enough?

Also what do you think about just putting `flat` on the top level element and just targeting it with `.flat thing`?